frontera/contrib/backends/hbase.py

```diff
@@ -189,9 +190,9 @@ def get_next_requests(self, max_n_requests, partition_id, **kwargs):
         limit = min_requests
         tries = 0
         count = 0
-        prefix = '%d_' % partition_id
+        prefix = b'%d_' % partition_id
```

Comment:
If we use `to_bytes` for converting prefix and filter(as I suggested above) we end up with filter = `PrefixFilter b'0_'` thus we get an invalid prefix error. To avoid this we can either not convert prefix, or not convert both prefix and filter. This won't be a problem since happybase accepts native_string as argument for filter.